src/pages/App.tsx

```diff
@@ -65,29 +65,18 @@ initializeAnalytics(ANALYTICS_DUMMY_KEY, OriginApplication.INTERFACE, {
   isProductionEnv: isProductionEnv(),
 })
 
-const AppWrapper = styled.div`
-  display: flex;
-  flex-flow: column;
-  align-items: flex-start;
-  height: 100%;
-`
-
 const BodyWrapper = styled.div`
   display: flex;
   flex-direction: column;
   width: 100%;
-  height: 100%;
-  padding: 72px 0px 0px 0px;
+  padding: ${({ theme }) => theme.navHeight}px 0px 5rem 0px;
   align-items: center;
   flex: 1;
-  ${({ theme }) => theme.deprecated_mediaWidth.deprecated_upToSmall`
-    padding: 52px 0px 16px 0px;
-  `};
```

Comment:
We do not need a different padding on mobile/desktop, because in all cases, the top navigation bar is the same (72px). 

Smaller padding was probably set by a mistake as a quick fix to overcome issues caused by a `MobilePopUp` (see PR description). Adding `20px` to `52px` makes up `72px`, so all makes sense!

This change makes it very explicit about paddings. 

Also, note that `5rem` was added to remove artificial `Marginer` component.